packages/module-app/src/enable-app-module.ts

```diff
@@ -0,0 +1,21 @@
+import type { IModulesConfigurator } from '@equinor/fusion-framework-module';
+import type { AppConfigBuilderCallback } from 'AppConfigBuilder';
+
+import { module } from './module';
+
+/**
+ * Method for enabling the Service module
+ * @param configurator - configuration object
+ */
+export const enableAppModule = (
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    configurator: IModulesConfigurator<any, any>,
+    builder?: AppConfigBuilderCallback
+): void => {
+    configurator.addConfig({
+        module,
+        configure: (appConfigurator) => {
```

Comment:
Will this work if no bulder is provided?
To me it looks like the configure function will be overwritten all times.

Reply:
if not provided, default functionality for builder will be executed and generate configuration to the framework (fusion services)